libs/test/SmartStoreTest/src/com/salesforce/androidsdk/smartstore/store/SmartStoreAlterTest.java

```diff
@@ -737,9 +737,8 @@ private void tryAlterSoupInterruptResume(AlterSoupLongOperation.AlterSoupStep to
             // Check last step completed
             Assert.assertEquals("Wrong step", toStep, ((AlterSoupLongOperation) operations[0]).getLastStepCompleted());
 
-            // Simulate restart (clear cache and call resumeLongOperations)
-            DBHelper.getInstance(db).clearMemoryCache();
-            store.resumeLongOperations();
```

Comment:
We are no longer calling resumeLongOperations in the test, instead we are closing and re-opening the database.